Sunset and sunrise SVG's for the weather_card's

coasting24 opened the pull request.

coasting24 committed:
Sunset and sunrise SVG's for the weather_card's

damianeickhoff reviewed (commented):
These files are for the future update of the weather card, right?

coasting24 commented:
Yes, sorry I should have mentioned that. But they can be pushed into any release as they are stored in the custom_icons folder.

coasting24 committed:
Merge branch 'damianeickhoff:main' into main

coasting24 committed:
Update hc_weather_card.yaml

Change the date format from US to local HA installation.

coasting24 committed:
Lightning weather condition

coasting24 committed:
Merge branch 'damianeickhoff:main' into main

coasting24 committed:
Merge branch 'damianeickhoff:main' into main

damianeickhoff committed:
Create CONTRIBUTING.md

deickhoff committed:
feat: Added issue templates

damianeickhoff committed:
Create release.yaml

Automate creating the release .zip file with the needed folders (dashboard, theme, www and custom_icons)

damianeickhoff committed:
Create auto_label_issues.yaml

Added a auto-label workflow

damianeickhoff committed:
Update auto_label_issues.yaml

damianeickhoff committed:
Update auto_label_issues.yaml

damianeickhoff committed:
Update improvement_request.md

damianeickhoff committed:
Update bug_report.md

damianeickhoff committed:
Rename improvement_request.md to enhancement_request.md

damianeickhoff committed:
Update feature_request.md

damianeickhoff committed:
Update auto_label_issues.yaml

damianeickhoff committed:
Create auto_label_issues_manually.yaml

damianeickhoff committed:
Update auto_label_issues_manually.yaml

damianeickhoff committed:
Update auto_label_issues_manually.yaml

damianeickhoff committed:
Update auto_label_issues_manually.yaml

damianeickhoff committed:
Update auto_label_issues_manually.yaml

damianeickhoff committed:
Delete .github/workflows/auto_label_issues_manually.yaml

coasting24 committed:
Merge branch 'damianeickhoff:main' into main

coasting24 committed:
Pouring Weather Condition
